pkgs/misc/lilypond/unstable.nix

```diff
@@ -0,0 +1,17 @@
+{ lib, fetchurl, guile, lilypond }:
+
+let
+
+  version = "2.23.11";
+
+in
+
+(lilypond.override {
+  inherit guile;
```

Comment:
As of now this gives us `guile_2_2`, right? Do we want to be explicit, maybe in `all-packages.nix`?

Reply:
Looks like no: https://github.com/NixOS/nixpkgs/pull/182614#discussion_r933495425